asyncapi/twilio/Module.md

```diff
@@ -1,31 +1,34 @@
 ## Overview
 
 The Twilio Trigger allows you to listen to Twilio SMS and Call status change events.
-1. Listen to incoming message events and message status change callback events from the twilio SMS.
-2. Listen to incoming call events and call status change callback events from the twilio Voice Call.
+1. Listen to incoming message events and message status change callback events from the Twilio SMS.
+2. Listen to incoming call events and call status change callback events from the Twilio Voice Call.
```

Comment:
This doesn't need to be an ordered list. So shall we make these bullet points? 
Suggestion:
The Twilio trigger allows you to listen to Twilio SMS and call status change events similar to the following:
- Incoming message events and message status change callback events from Twilio SMS.
- Incoming call events and call status change callback events from Twilio voice call.